src/python/test/test_dxpy.py

```diff
@@ -458,6 +458,17 @@ def test_upload_string_dxfile(self):
 
         self.assertTrue(filecmp.cmp(self.foo_file.name, self.new_file.name))
 
+    def test_upload_empty_dxfile(self):
+        # Checking default backend
+        dxpy.new_dxfile().close()
+        dxpy.upload_local_file(filename=self.new_file.name, project=self.proj_id, wait_on_close=True)
+        # Checking azure backend
+        with testutil.temporary_project("TestDXFile.test_upload_empty_dxfile azure", select=True, region="azure:westus") as ap:
+            with dxpy.DXFile(project=ap.get_id()) as f:
```

Comment:
Is the idea to have this be analogous to what's done in the default region above? If so, please create the file in the same way in both regions, just to make it clear that there's no difference. The shortest idiomatic ways are the following:

```
# (1) Use new_dxfile
with new_dxfile(project=...) as my_file:
    pass # Write an empty file
# (2) Use upload_string
my_file_2 = upload_string('', project=...)
```

Also, please verify (in at least one of the cases above) that the file size is 0 after the file is closed.